### PR TITLE
Ruelvla 12

### PR DESCRIPTION
**Pull request by beahuszar**
Hi Süvi, I have completed the DB structure, can we check it together today sometime? 🥇

### CODE REVIEW

File: rueppellii-lvlup-asp.net-core/Models/User.cs [hunk 1]
```diff
@@ -19,6 +19,8 @@ public class Users
 
         public string Pic { get; set; }
 
-        public List<Badges> Badges { get; set; }
+        public List<Review> Reviews { get; set; }
+        public List<Pitch> Pitches { get; set; }
+        public List<UserLevel> UserLevels { get; set; }
```
Comment (MichaelBarcley) [inline rueppellii-lvlup-asp.net-core/Models/User.cs:24]: According to the drawn DB scheme (https://app.sqldbm.com/MySQL/Share/m3vZyWVhNI3ZPAnWPSs_GEGFrngIE8md_DYjF4jNYw0) Users should have Badges not a combination of Reviews, Pitches and UserLevels. Is this smth I have missed and you agreed to do it like this?

Comment (MichaelBarcley) [reply rueppellii-lvlup-asp.net-core/Models/User.cs:24]: Resolved.

File: rueppellii-lvlup-asp.net-core/Models/Pitch.cs [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
         [MaxLength(200)]
         public string PitchMessage { get; set; }
 
-        public List<Reviews> Reviewers { get; set; }
-
-        public Users User { get; set; }
-        public Badges Badges { get; set; }
+        public List<Review> Reviewers { get; set; }
+        public User User { get; set; }
+        public Badge Badges { get; set; }
```
Comment (MichaelBarcley) [inline rueppellii-lvlup-asp.net-core/Models/Pitch.cs]: Shouldn't be this one called only "Badge" in a singular form, as a Pitch only has one Badge?

Comment (beahuszar) [reply rueppellii-lvlup-asp.net-core/Models/Pitch.cs]: done :)

**Review by vmikii — COMMENTED** [review]
0

**Review by vmikii — APPROVED** [review]

**Pull request merged**